test/ffprober_test.rb

```diff
@@ -2,31 +2,38 @@
 
 class FfproberTest < Minitest::Test
   def test_json_input
-    ffprobe = Ffprober::Parser.from_json(File.read("test/assets/sample video.json"))
+    ffprobe = Ffprober::Parser.from_json(
+      File.read('test/assets/sample video.json')
+    )
 
-    assert_equal "test/assets/sample video.m4v", ffprobe.format.filename
+    assert_equal 'test/assets/sample video.m4v', ffprobe.format.filename
 
-    assert_equal "44100", ffprobe.audio_streams.first.sample_rate
+    assert_equal '44100', ffprobe.audio_streams.first.sample_rate
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.